app/views/pwa/manifest.json.jb

```diff
@@ -0,0 +1,30 @@
+manifest = {
+  name: "Kaigi on Rails 2023 conference-app",
+  short_name: "Kaigi on Rails 2023",
```

Comment:
Too long? How do you think @okuramasafumi

Reply:
"Kaigi on Rails" seems enough when icon shows it's 2023.

Reply:
342d67f

Reply:
| iOS | Android |
| --- | --- |
| ![image](https://github.com/kaigionrails/conference-app/assets/4487291/9ec8bb6e-de81-4548-a658-97e8321434d0) | ![image](https://github.com/kaigionrails/conference-app/assets/4487291/85b0af08-5152-449c-9a6f-d46d7a1be0c7) |

looks good.